website/docs/r/guardduty_member.html.markdown

```diff
@@ -39,12 +41,23 @@ The following arguments are supported:
 * `account_id` - (Required) AWS account ID for member account.
 * `detector_id` - (Required) The detector ID of the GuardDuty account where you want to create member accounts.
 * `email` - (Required) Email address for member account.
+* `invite` - (Optional) The flag to send an invitation to specified AWS account ID.
+* `invitation_message` - (Optional) Message for invitation.
+
+## Timeouts
+
+`aws_guardduty_member` provides the following [Timeouts](/docs/configuration/resources.html#timeouts)
+configuration options:
+
+- `create` - (Default `60s`) How long to wait for a certificate to be issued.
```

Comment:
Copypasta 🍝  needs to be updated to match what the timeout is for with this specific resource 😄

Reply:
ohh... such a shame.... :( 
Sorry for the sloppy documentation...